src/cts/src/TreeBuilder.h

```diff
@@ -50,6 +50,29 @@ namespace utl {
 class Logger;
 }  // namespace utl
 
+// Hash and compare functions for occupiedLocations_ map
+// Use this hash table to keep track of occupied locations
+namespace std {
+template <>
+struct hash<cts::Point<double>>
```

Comment:
Could you just use the integer point like odb::Point?

Reply:
OK.  Let me look into it.